downloader_app/tiff_downloader.py

```diff
@@ -62,26 +68,30 @@
 
 
 def download_tiffs(source, dates, point1, point2, opt=False):
-    """ 
-    Function responsible for the main calls. 
-    
+    """
+    Function responsible for the main calls.
```

Comment:
for the first line of the docstring ... write it in an imperative form, for example:

`Download tiff files according to the given parameters.`